Source/DbgEngHelpers/DebugClient.cs

```diff
@@ -25,6 +25,19 @@ public static IDebugClient OpenDumpFile(string dumpFile, string symbolPath)
             return client;
         }
 
+        public static IDebugClient OpenProcess(string processPath, string processArguments, string symbolPath, uint debugEngineOptions)
```

Comment:
Fixed.

---
In reply to: [69626980](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69626980) [](ancestors = 69626980)